x-pack/plugins/lists/server/index.ts

```diff
@@ -9,6 +9,10 @@ import { PluginInitializerContext } from '../../../../src/core/server';
 import { ConfigSchema } from './config';
 import { ListPlugin } from './plugin';
 
+// exporting these since its required at top level in siem plugin
```

Comment:
`// export these since they are required at top levels within other plugin`

Since this can be used in other plugins outside of SIEM for other users of the plugin. I take it there was a linter rule violation without doing this?

Reply:
Thanks for looking at this - I took another look and I can remove the `ListPluginSetup` as I'm apparently able to import that from it's real location. The `ListClient` will have to remain exported here because the linter does complain about that one.

Reply:
Nevermind. I need to keep the ListPluginSetup as that started complaining again.